src/layouts/Settings/AnalyticsSettings.tsx

```diff
@@ -34,8 +34,23 @@ export const AnalyticsSettings = ({
         </FormControl>
 
         <FormControl isDisabled={isError}>
-          <FormLabel>Google Analytics</FormLabel>
-          <Input w="100%" {...register("ga")} />
+          <FormLabel>Google Analytics (UA)</FormLabel>
+          <SectionCaption label="" icon={BiInfoCircle}>
+            This field will be removed following the deprecation of Universal
+            Analytics on 1 July 2023.{" "}
+            <Link
+              href="https://support.google.com/analytics/answer/11583528?hl=en"
```

Comment:
should this url be in config/constants file?

Reply:
Sure, have moved all our external url calls to a separate constants file in https://github.com/isomerpages/isomercms-frontend/pull/1229/commits/8c706cd8f4b2d9e237e95d7c65abed4f673f45b6